contracts/CollectionPartyFactory.sol

```diff
@@ -60,43 +60,43 @@ contract PartyBuyFactory {
 
     function startParty(
         address _nftContract,
-        uint256 _tokenId,
         uint256 _maxPrice,
         uint256 _secondsToTimeout,
+        address[] calldata _deciders,
         Structs.AddressAndAmount calldata _split,
         Structs.AddressAndAmount calldata _tokenGate,
         string memory _name,
         string memory _symbol
-    ) external returns (address partyBuyProxy) {
+    ) external returns (address partyProxy) {
         bytes memory _initializationCalldata =
             abi.encodeWithSelector(
-            PartyBuy.initialize.selector,
+            CollectionParty.initialize.selector,
             _nftContract,
-            _tokenId,
             _maxPrice,
             _secondsToTimeout,
+            _deciders,
             _split,
             _tokenGate,
             _name,
             _symbol
         );
 
-        partyBuyProxy = address(
+        partyProxy = address(
             new NonReceivableInitializedProxy(
                 logic,
                 _initializationCalldata
             )
         );
 
-        deployedAt[partyBuyProxy] = block.number;
+        deployedAt[partyProxy] = block.number;
 
-        emit PartyBuyDeployed(
-            partyBuyProxy,
+        emit CollectionPartyDeployed(
```

Comment:
Would it be better to have all the contracts emit an event of the same name with a party type attribute?

Reply:
would be a question for @steveklebanoff based on what he prefers tbh

Reply:
but i am not opposed to the idea

Reply:
not a big deal either way -- should be quick for our graphql scraping code to handle either